sormas-ui/src/main/java/de/symeda/sormas/ui/task/TaskEditForm.java

```diff
@@ -241,6 +244,40 @@ protected void addFields() {
 		});
 	}
 
+	private List<DistrictReferenceDto> getCaseDistricts(CaseDataDto caseDto) {
+
+		if (caseDto.getResponsibleDistrict() != null && caseDto.getDistrict() == null) {
```

Comment:
This shouldn't be possible under any circumstances - is there a specific reason why you added this check?